__tests__/app.test.js

```diff
@@ -93,33 +93,88 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
 
-  describe("GET /api/articles", () => {
-    it("return array of articles with following properties", () => {
+describe("GET /api/articles", () => {
+  it("return array of articles with following properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const {
+          body: { articles },
+        } = response;
+        expect(Array.isArray(articles)).toBe(true);
+        expect(articles).toBeSorted({ descending: true });
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
+          expect(article).toHaveProperty("comment_count");
+        });
+      });
+  });
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("returns article that matches id", () => {
       return request(app)
-        .get("/api/articles")
+        .get("/api/articles/1/comments")
         .expect(200)
         .then((response) => {
           const {
             body: { articles },
           } = response;
-          expect(Array.isArray(articles)).toBe(true);
-          expect(articles).toBeSorted({ descending: true });
           articles.forEach((article) => {
+            expect(Array.isArray(articles)).toBe(true);
+            expect(article).toHaveProperty("comment_id");
+            expect(article).toHaveProperty("votes");
+            expect(article).toHaveProperty("created_at");
             expect(article).toHaveProperty("author");
-            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("body");
             expect(article).toHaveProperty("article_id");
-            expect(article).toHaveProperty("topic");
-            expect(article).toHaveProperty("created_at");
-            expect(article).toHaveProperty("votes");
-            expect(article).toHaveProperty("article_img_url");
-            expect(article).toHaveProperty("comment_count");
+          });
+        });
+    });
+    it("check property types of values", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((response) => {
+          const {
+            body: { articles },
+          } = response;
+          articles.forEach((article) => {
+            expect(typeof article["comment_id"]).toBe("number");
+            expect(typeof article.votes).toBe("number");
+            expect(typeof article["created_at"]).toBe("string");
+            expect(typeof article.author).toBe("string");
+            expect(typeof article.body).toBe("string");
+            expect(typeof article["article_id"]).toBe("number");
```

Comment:
These first two tests could be condensed into a single test. Again, check the jest docs